index.html

```diff
@@ -59,11 +58,151 @@ <h1>I’m Alexandra.   Glad   to  see      you!</h1>
 
 
                 </div>
-              </div>
+              </div>        
+
+              <section class="mywork">
+                    <article class="myproject">
+                        <div class="pheader">
+                            <img src="./img/projectone.png"> 
+                                                      
+                        </div>
+                        <div class="ptitle">
+                            <h3 >Tonic </h3>
+                        </div>                        
+                        <uL class="details">                           
+                                
+                                <li class="conpoy"> CANOPY</li>                             
+                                <li class="backeddev">Back End Dev</li>                               
+                                <li class="text2015">2015</li> 
+                            </uL>                           
+                        
+                        <p calss="everyday">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <div class="lower_part">
+                            <ul class="projectstack">
+                                <li><a>HTML</a></li>
+                                <li><a>CSS</a></li>
+                                <li><a>Javascript</a></li>
+                            </ul>
+                            <a class="button">See project</a>
+                        </div>
+                    </article>
```

Comment:
You have done a great job implementing all the elements from the Figma design. 👍🏿  
- [ ] However, please make sure that your designs are similar to the Figma designs as requested by your previous reviewers. This will go a long way to help you stick to client preferences and satisfaction in your future job 🧰
 
Please compare your sedign below to the figma template provided:

![image](https://github.com/samuelkinyanjui/Portfolio-mobile-first/assets/117144028/a8e35b53-bcd9-43a4-a0bf-5a235ae2c863)